services/ark-api/.env.example

```diff
@@ -0,0 +1,21 @@
+# ARK API Environment Configuration
+# Copy this file to .env and modify as needed
+
+# Authentication Configuration
+# Set to "true" to completely disable authentication for all routes
+# WARNING: Only use this in development or testing environments
+ARK_SKIP_AUTH=false
+
+# OKTA JWT Token Validation
```

Comment:
Although this is not a leak it really absolutely should not be here, please be extremely mindful of config like this